style.css

```diff
@@ -0,0 +1,94 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+a:hover {
+  color: #172b4d;
+}
+
+.html.body {
+  font-family: Poppins, sans-serif;
+}
+
```

Comment:
[OPTIONAL] 👍🏽 
- I think it will be better to add your `font-family: Poppins, sans-serif;` to the `body` style, instead of `.html.body` as Poppins is currently not rendered on your portfolio. 🌸 

Apparently, this is what I mean 👇🏽 
```
body {
  font-family: Poppins, sans-serif;
}
```

What do you think about this?? It should reflect after you have done this.🤗

Reply:
It's a great idea I put it into use